libs/ktxreader/src/Ktx1Reader.cpp

```diff
@@ -83,9 +83,11 @@ Texture* createTexture(Engine* engine, const Ktx1Bundle& ktx, bool srgb,
 
     auto texformat = toTextureFormat(ktxinfo);
     if (srgb && !isSrgbTextureFormat(texformat)) {
-        utils::slog.w << "Requested sRGB format but KTX contains a linear format. ";
+        utils::slog.w << "Requested sRGB format but KTX contains a linear format. "
+            << utils::io::endl;
     } else if (!srgb && isSrgbTextureFormat(texformat)) {
-        utils::slog.w << "Requested linear format but KTX contains a sRGB format. ";
+        utils::slog.w << "Requested linear format but KTX contains a sRGB format. "
+            << utils::io::endl;
```

Comment:
I feel like logs like this should be on debug builds only.

Either that or this condition is an error, and preconditions are violated. I guess... is this a misuse of the API? If so, I think it should be documented and we should use ASSERT_PRECONDITION and method should not be `noexcept`.

If we want to allow it, but warn that it's wrong, I think DEBUG build only is more appropriate.